### PR TITLE
Use click to generate Packt CLI

### PR DESCRIPTION
**Pull request by mjenczmyk**
I've prepared a modification that allows to install script via `pip` and makes cloning this repository to use it unnecessary. It have been tested as cron job on Debian mashine for some months.

For testing purposes install it with `pip install git+https://github.com/mjenczmyk/Packt-Publishing-Free-Learning.git@setup` instead of with the command given in README.

Unless we no longer want to support Windows somebody should check if that works on Windows (I don't have access to any mashine with Windows).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,199 +1,179 @@
...
-  then run the following command:
+### Installation
 
-  ```  
-  python get-pip.py
-  ```
-* Optionally install [*virtualenv*](http://docs.python-guide.org/en/latest/dev/virtualenvs/) (pip install virtualenv)
+To install script simply run
+```
+pip install git+https://github.com/igbt6/Packt-Publishing-Free-Learning.git@master
```
Comment (mjenczmyk) [inline README.md]: Actually for testing this feature before merging you should use `pip install git+https://github.com/mjenczmyk/Packt-Publishing-Free-Learning.git@setup`

**Comment by mjenczmyk** [comment]
@igbt6 Any thoughts about this?

**Comment by mjenczmyk** [comment]
May I remind you about the pull request? It's about a month and a half now...

**Comment by igbt6** [comment]
@mjenczmyk  hey, yeah, still no time for checking, too many task day by day,  Somehow in Novemebr i will check all your PRs : ) 

Regards!

**Comment by mjenczmyk** [comment]
Thanks, I won't bother you again than - take you time.

**Comment by mjenczmyk** [comment]
Any progress with this code review?

**Comment by igbt6** [comment]
Some conflicts and we should test it on Windows, Are you sure it will work ?

**Comment by mjenczmyk** [comment]
I've rebased my branch with your `master` - there are no conflicts anymore.

Yep, it works - it generates `packt-cli` command when installed, so there won't be any need to clone the repository if you don't need to change any code. You can still clone the repository, modify the code and install it into virtualenv by running `pip install -e .`.

The command will be generated regardless of user's OS, but I'm not sure how to setup `schtasks` on Windows (but it can be done, one way is to install this package globally, but I wouldn't like to recommend such solution in `README.md` as it pollutes your Python environment).

In short - it will work, but the scheduled tasks setup description will be outdated, if somebody can do it (do you have access to Windows mashine?) I'd be grateful. If we don't have anybody to check it that's worse - but that also means that we are no longer able to support this script for Windows users.
